hatchet_sdk/contracts/events_pb2.pyi

```diff
@@ -1,13 +1,27 @@
-from google.protobuf import timestamp_pb2 as _timestamp_pb2
```

Comment:
Are these stubs autogenerated? I was assuming running the formatters over them is fine, but just wanted to confirm

Reply:
they are, probably answers my q above.